<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomScheduling</class>
 <widget class="QDialog" name="CustomScheduling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Instant Schedule</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>350</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="shiftCHeck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="dailyCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Dialy</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="weeklyCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Weekly</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="monthlyCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Monthly</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateFrom">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>194</width>
     <height>27</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>74</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>74</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>To</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>431</width>
     <height>231</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Shift</string>
    </attribute>
    <widget class="QTimeEdit" name="shiftTime1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="shiftTime2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QTimeEdit" name="shiftTim3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ShiftXLStemplate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Br</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mon</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>170</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tue</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Wed</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Thu</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Fri</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_11">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sat</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_12">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sun</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XLS template</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="shiftSubj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="shiftAttach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attachment Name</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shiftAttachXLS">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach XLS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="shiftAttachCSV">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach CSV</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Daily</string>
    </attribute>
    <widget class="QCheckBox" name="dailyWed">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Wed</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyTue">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>170</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Tue</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dailySubj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailySun">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sun</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attachment Name</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyAttachCSV">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach CSV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyAttachXLS">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach XLS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyMon">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>51</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Mon</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="dailyTime1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Br</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailySat">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sat</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dailyXLSTemplate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XLS template</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyFri">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Fri</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="dailyThu">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>170</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Thu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dailyAttach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Weekly</string>
    </attribute>
    <widget class="QTimeEdit" name="weeklyTime1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="weeklyAttachXLS">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach XLS</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="weeklyAttachCSV">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach CSV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attachment Name</string>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Br</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="weeklySubj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="weeklyAttach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="weeklyXLStemplate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XLS template</string>
     </property>
    </widget>
    <widget class="QComboBox" name="weeklyDay">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <property name="maxVisibleItems">
      <number>10</number>
     </property>
     <item>
      <property name="text">
       <string>Monday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Thursday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Friday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Saturday</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sunday</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Monthly</string>
    </attribute>
    <widget class="QCheckBox" name="monthlyXLS">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>130</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach XLS</string>
     </property>
    </widget>
    <widget class="QTimeEdit" name="monthlyTime1">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>70</y>
       <width>118</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="monthlyCSV">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>100</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Attach CSV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Attachment Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="monthlySubject">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_5">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Br</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="monthlyAttach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="monthlyXLSTemplate">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>XLS template</string>
     </property>
    </widget>
    <widget class="QComboBox" name="monthlyDays">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">combobox-popup: 0;</string>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>16</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>17</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>18</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>26</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>27</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>28</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>29</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="parametersCheck">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Parameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>toolButton</tabstop>
  <tabstop>dailyCheck</tabstop>
  <tabstop>weeklyCheck</tabstop>
  <tabstop>monthlyCheck</tabstop>
  <tabstop>dateFrom</tabstop>
  <tabstop>dateTo</tabstop>
  <tabstop>ShiftXLStemplate</tabstop>
  <tabstop>shiftCHeck</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CustomScheduling</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CustomScheduling</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
